compositor_render/src/wgpu/texture/nv12.rs

```diff
@@ -31,15 +31,15 @@ impl<'a> NV12TextureView<'a> {
         let view_y = texture.create_view(&wgpu::TextureViewDescriptor {
             label: Some("y plane nv12 texture view"),
             dimension: Some(wgpu::TextureViewDimension::D2),
-            format: Some(wgpu::TextureFormat::NV12),
+            format: Some(wgpu::TextureFormat::R8Unorm),
```

Comment:
Why? This is literally called "NV12TextureView"?

Reply:
That's because  you can't view NV12 textures as NV12 in the shader. If you want to use them in a shader, you have to attach the Y plane as a plane with one color (R8Unorm) and the UV plane as a plane with two colors (Rg8Unorm).